daemon/cmd/fqdn.go

```diff
@@ -318,13 +318,15 @@ func (d *Daemon) notifyOnDNSMsg(lookupTime time.Time, ep *endpoint.Endpoint, epI
 	if msg.Response {
 		flowType = accesslog.TypeResponse
 		addrInfo.DstIPPort = epIPPort
-		addrInfo.DstIdentity = ep.GetIdentity()
+		addrInfo.DstEPID = ep.GetID()
+		addrInfo.DstSecIdentity, _ = ep.GetSecurityIdentity()
```

Comment:
Would you mind adding a quick comment explaining why it's okay that we can avoid checking the error here? It'll probably save folks some time in the future.